app/src/main/java/nerd/tuxmobil/fahrplan/congress/designsystem/colors/ColorScheme.kt

```diff
@@ -58,9 +58,33 @@ data class ColorScheme(
     val onTertiaryFixedVariant: Color,
 
     // Custom colors
-    val topAppBarContainer: Color,
+    val listItem: Color,
     val divider: Color,
     val searchBarDivider: Color,
+    val sessionListHeaderDayDate: Color,
+    val textLink: Color,
+    val topAppBarContainer: Color,
+    val topAppBarTitleText: Color,
+    val topAppBarNavigationIcon: Color,
+    val topAppBarIcon: Color,
```

Comment:
Where is this used? I see there's also `topAppBarNavigationIcon`. So this is for action icons? Maybe `topAppBarActionIcon`?